services/distribution/src/main/java/app/coronawarn/server/services/distribution/utils/SerializationUtils.java

```diff
@@ -0,0 +1,34 @@
+package app.coronawarn.server.services.distribution.utils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.type.TypeFactory;
+import java.util.function.Function;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class SerializationUtils {
+
+  private static final Logger logger = LoggerFactory.getLogger(SerializationUtils.class);
+
+  private SerializationUtils() {
+  }
+
+  /**
+   * Deserialize Json string into provided function type object.
```

Comment:
jdoc suggestion : "Deserialize json string into an object of type T. The type must also be provided to the underlying Jackson library in the form of a JavaType supplied by the function parameter."